Source/Shift/Classes/SplitTransition.swift

```diff
@@ -88,19 +102,132 @@ public class SplitTransition: NSObject {
     }()
 
     /**
+     * In an interactive transition, splitOffset
+     * determines the vertical distance of the initial
+     * split when the cell is first tapped
+     */
+    public var splitOffset: CGFloat = 0.0
+
+    /**
+    * Scroll distance in UI points
+    */
+    var interactiveTransitionScrollDistance: CGFloat = 0.0
```

Comment:
I prefer to lean on the type inference system and use constructor syntax in these cases:

```swift
var interactiveTransitionScrollDistance = CGFloat(0.0)
```

Good use of the `0.0` syntax though, I find that things can get confusing if you aren't a little specific about your literals.